jump-core/src/main/java/com/bitdecay/jump/level/builder/LevelBuilder.java

```diff
@@ -313,6 +313,12 @@ public void selectObject(BitPointInt startPoint, boolean add) {
 				}
 			}
 		}
+		if (spawn != null) {
+			// This is gross. I don't like.
+			if (spawn.rect.xy.minus(startPoint).len() < SpawnObject.OUTER_DIAMETER) {
+				selection.add(spawn);
+			}
+		}
 	}
```

Comment:
This was basically making it so the user can select the spawn point, therefore able to change what stats the player spawns at it with.